about.html

```diff
@@ -0,0 +1,178 @@
+          </p>
+        </div>
+        <div class="contact-us">
+          <p>Please Contact us through Email or Calls to book for an appointment</p>
+          <p>At Spectra Beauté</p>
+          <span>nkargleeny@gmail.com</span>
+        </div>
+      </section>
+
+      <section class="about-logo">
+        <h2>Beauty, Wellness, Inspiration and Wellbeing Only at Spectra</h2>
+        <div class="line"></div>
+        <p class="about-logo-desc">
+            Spectra Beauty is a high end Nail, Beauty & Hair clinic dedicated to quality services. Brow Bar, Lash extensions, Massage, Facial, Hair extensions.
+        </p>
+        <div class="intro-about-desc">
+          <div class="footer-logo">
+            <img class="footer-img" src="./Images/Spectra-logo.webp" alt="Spectra-logo" />
+          </div>
+        </div>
+      </section>
+      
+      <div><hr class="divider"></div>
+      <section class="about-started">
+        <h2>Our Activities</h2>
+        <div class="line"></div>
+        <p>
+          Spectra Beauté has existed for Ten years now. And the journey has being full of growth creativity and inspiration.
+        </p>
+        <div class="activities" id="sum">
+          <div class="activity">
+            <img src="./Images/Spec-store.JPG" class="spec-img" alt="started1">
+            <h2>2019</h2>
+            <p>Spectra Professional Our Store</p>
+          </div>
+          <div class="activity">
+            <img src="./Images/Spec-team.JPG" class="spec-img" alt="Team-spectra" />
+            <h2>2022</h2>
+            <p>Without Team Spectra there's no Spectra</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="partners-section1">
+        <h2>Partners</h2>
+        <div class="line"></div>
+        <ul class="partners">
+          <li>
+            <h3><img src="./Images/Partner1.JPG" alt="Partner1" id="mr-5" class="partner1"></h3>
+          </li>
+          <li>
+            <h3><img src="./Images/partner2.png" alt="Partner2" class="partner"></h3>
+          </li>
+          <li>
+            <h3><img src="./Images/partner3.png" alt="Partner3" class="partner"></h3>
+          </li>
+          <li>
+            <h3><img src="./Images/partner4.png" alt="Partner4" class="partner3"></h3>
+          </li>
+        </ul>
+      </section>  
+      
+      <footer id="foot">
+        <div class="footer-logo">
+          <img class="footer-img1" src="./Images/Spectra-logo.webp" alt="Spectra logo">
+        </div>
+        <div class="footer-message-msg">
+          <p>@Spectra Beauté will always be there for you to give you the peace and comfort you deserve</p>
+          <p>A source od inspiration and wellbeing Spectra Beauté</p>
+        </div>
```

Comment:
- Please make sure your design matches that of the original design as seen below.
Your design | Behance design
![screenshot (2)](https://user-images.githubusercontent.com/44624138/181957515-abe4bb0c-6d29-4e02-80e9-153d5a4c4e59.png) | ![screenshot (4)](https://user-images.githubusercontent.com/44624138/181958467-3a89cf33-ce54-430c-969c-7924339aac28.png)

Reply:
- Same with the home page 😄